### ISSUE TITLE
Exporting static does not work.

### ISSUE THREAD

**Issue by jackpordi** [pablopunk/nextjs-redirect#24]
There seems to have been changes since #4 that broke this feature. To be specific, it's that `getInitialProps` now has calls to `Router` again once more.

```typescript
    static async getInitialProps({ res, query }) {
      if (res?.writeHead) {
        let url = redirectUrl

        if (options?.params === true) {
          const param = redirectUrl
          if (!query[param]) {
            throw new Error(PARAMS_ERROR)
          }
          url = query[param]
        }

        res.writeHead(options?.statusCode ?? 301, { Location: url })
        res.end()
      } else if (options?.asUrl) {
        Router.push(redirectUrl, options.asUrl, { shallow: true })
      } else {
        Router.push(redirectUrl)
      }

      return {}
    }
```

**Comment by pablopunk** [comment]
hi @jackpordi ! I'm so sorry I missed this, my GitHub notifications are just a sinkhole...

You're right, I broke it while fixing other stuff. I'll push a fix now to make exports work again

**Comment by pablopunk** [comment]
@jackpordi It should be fixed on `6.0.1` 🎉
